content/legal.rst

```diff
@@ -127,7 +127,7 @@ Terms and Conditions
 +--------------------------------------------------------------------+-----------------------------------------------------------------------+------------------------------------------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------+
 |                                                                    | **English**                                                           |  Français                                                              | Nederlands                                                            | Deutsch                                                               | Español                                                               |
 +====================================================================+=======================================================================+========================================================================+=======================================================================+=======================================================================+=======================================================================+
-| Odoo Enterprise Agreement [#ltoe1]_                                | |view_enterprise_en| |download_enterprise_en|                         | |view_enterprise_fr| |download_enterprise_fr|                          | |missing_txt| |missing_pdf|                                           | |missing_txt| |missing_pdf|                                           | |missing_txt| |missing_pdf|                                           |
+| Odoo Enterprise Agreement [#ltoe1]_                                | |view_enterprise_en| |download_enterprise_en|                         | |view_enterprise_fr| |download_enterprise_fr|                          | |missing_txt| |missing_pdf|                                           | |missing_txt| |missing_pdf|                                           | |view_enterprise_es| |download_enterprise_es|                         |
```

Comment:
@Lau-Odoo 

Unless I'm mistaken, when you put this PR back to draft, it's because the pdf file (see lines 105-107) is pinging a pdf of v8 of the ES agreement. While your RST modification is updating from 8.0 -> 9.0 .
Here is the external link, still v8 from 2018 https://www.odoo.com/documentation/14.0/odoo_enterprise_agreement_es.pdf

Also, these PDF files don't seem to be maintained from the documentation repository. @odony @mart-e do you know why it is so and where they can be updated? Who is creating this PDF? Or can we at least go forward and merge this PR even if the PDF isn't up to date?

Also, shouldn't this PR be targeting v13.0 rather than 14.0? If so, you'll have to close this PR and create a new Pull Request as rebasing and changing the target on the PR would ping all code owners of this repo and that would suck big time ^^

Thank you very much :)